pkgs/by-name/te/tenderly-cli/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  fetchFromGitHub,
+  lib,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tenderly";
+  version = "1.6.5";
+
+  src = fetchFromGitHub {
+    owner = "Tenderly";
+    repo = "tenderly-cli";
+    tag = "v${version}";
+    hash = "sha256-7l6Od/U8vVZ1oKlmvMAUBQIT78aYwf7q3IjTbpmw1Nk=";
+  };
+
+  doCheck = false;
```

Comment:
why? either enable them or put why as a comment in the file

Reply:
idk I think there's something wrong with `lix` right now because it took 5 or 10 tries to build with tests enabled. built first try with them disabled.

Reply:
Try using it with nix

Reply:
Worked, but I think because it was cached. Not sure how to clear cache to rebuild fully

Reply:
`--rebuild` flag

Reply:
It's taking 3 or 4 tries to build here using `nix (Nix) 2.24.12`, but it built

Reply:
Sorry, I don't know how to do that. How can I disable individual tests?